pkg/sql/types/types.go

```diff
@@ -1011,7 +1011,7 @@ func (t *T) Name() string {
 	case BoolFamily:
 		return "bool"
 	case BytesFamily:
-		return "bytes"
+		return "bytea"
```

Comment:
what is `bytea` ?

Reply:
Postgres byte array data type. Cockroach treats it as an alias for 'bytes'

Postgres docs: https://www.postgresql.org/docs/current/datatype-binary.html

Cockroach DB docs: https://www.cockroachlabs.com/docs/stable/bytes.html